TheRealGD/therealgd
# Issue 86 - Submission and comment violation reporting

This PR allows a logged in user to report both submissions or comments. As per the issue, no report "reason" is required.

To test this issue:
- Log in as a normal user
- Click the "Report" link which is displayed underneath each submission.
- Click "Yes"/"Ok" to the prompt.
- Log out of the account and log in with a mod/admin
- Navigate to the mod forum for the forum that the submission was in
- You should see a "Submission Report" that links to the submission that was reported.

- Log in as a normal user
- Navigate to a submission with comments (or create one)
- Click the "Report" link which is displayed underneath each comment.
- Click "Yes"/"Ok" to the prompt.
- Log out of the account and log in with a mod/admin
- Navigate to the mod forum for the forum that the submission was in
- You should see a "Comment Report" that links to the comment that was reported.
